components/ui-api-layer/docs/proposals/exceed-resource-quota.md

```diff
@@ -0,0 +1,126 @@
+# Checking when resource quota has been exceeded proposal
```

Comment:
`Check when the ResourceQuota is exceeded`

Reply:
Done